tests/rptest/utils/rpenv.py

```diff
@@ -23,3 +23,19 @@ def sample_license():
         assert is_ci == "false"
         return None
     return license
+
+
+class IsCIOrNotEmpty:
+    """
+    Comparison with this object is true if the environemnt is CI, or if the
+    other value is not empty.
+
+    Useful as a markup on a test that must run in CI or when the env is present:
+
+    @env(REQUIRED_ENV=IsCIOrNotEmpty())
+    """
+    def __init__(self):
+        self.is_ci = os.environ.get("CI") == "true"
```

Comment:
not trying to belabor the point but just in case something/someone messes with the case of the value maybe this:

```suggestion
        self.is_ci = os.environ.get("CI").lower() == "true"
```

Reply:
I think all the other uses of "CI" assume uppercase.

Reply:
Well, maybe not: https://github.com/redpanda-data/redpanda/commit/8ca158eee25be28896fd59fb096c38a02bdc2893